README.md

```diff
@@ -10,18 +10,17 @@ Obtain an access TOKEN from [stattleship.com](www.stattleship.com). Load R and i
 
 ```
 library(stattleshipR)
-TOKEN <- 'insert-your-token-here'
+setToken('insert-your-token-here')
 ```
 
 Get all NBA players:
 
 ```
-library(stattleshipR)
 league = "nba"
 sport = "basketball"
 ep = "players"
 q_body = list()
-players = stattle(TOKEN, sport=sport, league=league, ep=ep, query=q_body, version=1, verbose=TRUE, walk=TRUE)
+players = stattle(sport=sport, league=league, ep=ep, query=q_body, version=1, verbose=TRUE, walk=TRUE)
 players_df = do.call("rbind", lapply(players, function(x) x$players))
```

Comment:
Would you even not make a data frame from the results?

If no, then maybe the method should return a data frame -- so no need for 2nd line?